src/components/lns/lns_input.vue

```diff
@@ -83,23 +87,41 @@ export default {
       this.$refs.form.setRecord(record);
       this.updating = true;
     },
+    startRenewing(record) {
+      this.renewing = true;
+      // set the type such that we default to one year
+      let renewRecord = {
+        ...record,
+        type: "lokinet_1y"
+      };
+      this.$refs.form.setRecord(renewRecord);
+    },
     onSubmit(record, oldRecord) {
       if (this.updating) {
         this.update(record, oldRecord);
+      } else if (this.renewing) {
+        this.renew(record);
       } else {
         this.purchase(record);
       }
     },
     onClear() {
       this.$refs.form.reset();
       this.updating = false;
+      this.renewing = false;
     },
     async update(record, oldRecord) {
       // Make sure we have a diff between the 2 records
-      const isOwnerDifferent = record.owner !== "" && record.owner !== oldRecord.owner;
-      const isBackupOwnerDifferent = record.backup_owner !== "" && record.backup_owner !== oldRecord.backup_owner;
+      const isOwnerDifferent =
+        record.owner !== "" && record.owner !== oldRecord.owner;
+      const isBackupOwnerDifferent =
+        record.backup_owner !== "" &&
+        record.backup_owner !== oldRecord.backup_owner;
       const isValueDifferent = record.value !== oldRecord.value;
-      const different = isOwnerDifferent || isBackupOwnerDifferent || isValueDifferent;
+      const different =
+        isOwnerDifferent || isBackupOwnerDifferent || isValueDifferent;
+
+      // TODO: This is a little confusing, if nothing was actually updated
```

Comment:
Does seem confusing, would there be anything we can do? I made it show success so users that clicked update are sure it worked

Reply:
maybe a yellow "nothing updated" snackbar?

Reply:
Might be confusing to the user 🤔 . Maybe disable the update button if nothing was different?

Reply:
Lol yeah, that's actually much better

Reply:
Addressed.